development/Vagrantfile.example

```diff
@@ -1,20 +1,72 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-BOX = ENV.fetch('BOX', 'ubuntu-12.04')
+require 'pry'
 
-Vagrant.require_plugin 'vagrant-proxyconf'
+BOX = ENV.fetch('BOX', 'ubuntu/bionic64')
+DISABLE_SHARE = false
+ENABLE_PROXY = false
+
+DOCKER_SOCKET="/var/run/docker.sock"
+SUPPORTS_DOCKER_IN_DOCKER = File.exists?(DOCKER_SOCKET)
+
+PROXY_HOST = ENV['PROXY_HOST'] || '127.0.0.1'
```

Comment:
To be constant with `BOX`, `ENV.fetch` could be used also here. Or use this `||` style also there.

Reply:
Good catch, i'll fix this too.

Reply:
Resolved via: 6f14c17